tests/datatypes/test_frozenset.py

```diff
@@ -37,6 +37,60 @@ def test_creation(self):
 
             """)
 
+    def test_contains(self):
+        # Normal Contains
+        self.assertCodeExecution("""
+            x = frozenset()
+            print(x)
+
+            x = frozenset({'a'})
+            print('a' in x)
+
+            y = {1,2}
+            x = frozenset(y)
+            print(2 in x)
```

Comment:
For each case, could there be something that tests both true/false cases, just to make sure they're working correctly? So for this one, perhaps `2` and `3`, or something like that.

(Also, is there much difference from the 'a' case above?)